src/surbhi/Assignment9Banking.java

```diff
@@ -0,0 +1,69 @@
+package surbhi;
+public class Assignment9Banking {
+
+	int creditCnt;
+	int debitCnt;
+	double currentBalance;
+	static int totalCreditCnt;
+	static int totalDebitCnt;
+	static int totalPrintBalCnt;
+
+	void setCurrentBal(double currentBal) {
+		currentBalance = currentBal;
+	}
+
+	void creditAmount(int creditAmount) {
+		currentBalance = currentBalance + creditAmount;
+		creditCnt++;
+		totalCreditCnt++;
+	}
+
+	void debitAmount(int debitAmountt) {
+		currentBalance = currentBalance - debitAmountt;
```

Comment:
here, you should add condition,
if (debitAmount < currentBalance)
then only deduct amount else print insufficient balance.